src/docfx/lib/HrefUtility.cs

```diff
@@ -80,15 +80,16 @@ public static string MergeHref(string targetHref, string sourceQuery, string sou
         /// Get the git remote information from remote href
         /// </summary>
         /// <param name="remoteHref">The git remote href like https://github.com/dotnet/docfx#master</param>
-        public static (string remote, string refspec) SplitGitHref(string remoteHref)
+        public static (string remote, string refspec, bool hasRefSpec) SplitGitHref(string remoteHref)
```

Comment:
Can check from caller
```suggestion
        public static (string remote, string refspec) SplitGitHref(string remoteHref)
```